content/en/blog/demystifying-fission-pool-manager.md

```diff
@@ -0,0 +1,171 @@
++++
+title = "Demystifying Fission - Pool Manager"
+date = "2022-01-28T11:30:34+05:30"
+author = "Atulpriya Sharma"
+description = "Learn how you can use Pool Manager to supercharge your Fission functions."
+categories = ["Fission"]
+type = "blog"
+images = ["images/featured/opentelemetry-datadog-fission-featured.png"]
++++
+
+Developers who we think are involved in just coding in fact do a lot of work - *integrating their code, setting up the infrastructure for running it, building and deploying their code*. But as the demand for faster shipping increased, people started to find ways to make this entire process agile.
```

Comment:
@techmaharaj can we please reduce this preamble ? Ideally we should wrap it in 300 characters.